Source/WebKit/UIProcess/haiku/WebPageProxyHaiku.cpp

```diff
@@ -53,9 +53,10 @@ String WebPageProxy::standardUserAgent(const String& applicationNameForUserAgent
     else
         osVersion = "Unknown";
 
-    standardUserAgentString = "Mozilla/5.0 (" + platform + "; " + osVersion + ") AppleWebKit/" + version
-        + " (KHTML, like Gecko) Version/5.0 Safari/" + version;
-
+    /*standardUserAgentString = "Mozilla/5.0 (" + platform + "; " + osVersion + ") AppleWebKit/" + version
```

Comment:
Please restore this, and fix the parameters instead to get the same, instead of hardcoding the values.